### DISCUSSION TITLE
False duplicate when re-uploading photo

### DISCUSSION THREAD

**Discussion by dbo999** [immich-app/immich#11154] [Q&A]
### The bug

When you delete a picture from Immich through web UI and then try to re-upload that same picture , it still says “Duplicate”.

### The OS that Immich Server is running on

Debian

### Version of Immich Server

v1.108.0

### Version of Immich Mobile App

v1.108.0

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
Irrelevant
```

### Your .env content

```Shell
Irrelevant
```

### Reproduction steps

```bash
1. Upload any picture.
2. Delete that same picture through web UI.
3. Upload same picture again.

It should say “Duplicate” and stop uploading the photo.
```

**Comment by bo0tzz** [comment]
Did you remove the file from the trash?

**Comment by dbo999** [comment]
Thanks! I would never look into trash...

**Comment by stratacast** [comment]
This is a bit of a silly "feature". Duplicate checking should not happen in the trash. The file is there because it is unwanted. In my case, Immich CLI corrupted a handful of photos so I am deleting images and reuploading them. I never would have thought to look in the trash for a duplicate.
